Source/WebCore/page/WindowFeatures.h

```diff
@@ -39,24 +39,31 @@ namespace WebCore {
 class FloatRect;
 
 struct WindowFeatures {
+    bool wantsPopup { false };
```

Comment:
Do we need a data member, can this be a member function like the other `wants*()`? It seems it gets computed based on the existing data members anyway.

Reply:
I can certainly implement it like that yes.